storm-core/src/jvm/backtype/storm/security/auth/authorizer/SimpleACLAuthorizer.java

```diff
@@ -96,7 +101,10 @@ public boolean permit(ReqContext context, String operation, Map topology_conf) {
         }
 
         if (_userCommands.contains(operation)) {
-            return true;
+            if (_nimbusUsers.size() > 0 && _nimbusUsers.contains(user))
+                return true;
+            else if (_nimbusUsers.size() == 0)
+                return true;
```

Comment:
I personally think 
```
return _nimbusUsers.size() == 0 || _nimbusUsers.contains(user);
```
is more readable than what is here, but this is really minor.